TicTacToe/Program.cs

```diff
@@ -9,238 +10,292 @@ namespace TicTacToe
         {
-            char[][] board = new char[10][];
-            char player1 = 'X';
-            char player2 = 'O';
-            char currentPlayer = 'X';
-            int z = 9;
+            char[][] board = new char[numOfBoards][];
+            char currentPlayer = player2;
+            int z;
 
-            buildBoard();
-            initBoard(board);
+            HomePage();
+            InitBoard(board);
+            BuildBoard(board);
 
-            do
+            while (!SubGameOver(cellsNumber, board, currentPlayer)) 
             {
+                currentPlayer = ChangePlayer(currentPlayer);
                 if (currentPlayer == player1)
                 {
-                    z = humanTurn(board, player1);
+                    z = HumanTurn(board, player1);
                 }
                 else
                 {
-                    z = computerTurn(board, player2);
+                    z = ComputerTurn(board, player2);
                 }
-                updateBoard(board);
-                subGameOver(z, board, currentPlayer);
-                currentPlayer = changePlayer(currentPlayer);
+                SubGameOver(z, board, currentPlayer);
+                BuildBoard(board);
             }
-            while (!subGameOver(9, board, currentPlayer));
 
             Console.Clear();
-            if (currentPlayer == player1)
+            if (Draw(cellsNumber, board))
+            {
+                DrawArt();
+            }
+            else if (currentPlayer == player1)
             {
-                humanWon();
+                HumanWon();
             }
             else
             {
-                humanLose();
+                HumanLose();
             }
         }
-        private static char changePlayer(char currentPlayer)
+        private static char ChangePlayer(char currentPlayer)
         {
-            if (currentPlayer == 'X')
+            if (currentPlayer == player1)
             {
-                currentPlayer = 'O';
+                currentPlayer = player2;
                 return currentPlayer;
             }
             else
             {
-                currentPlayer = 'X';
+                currentPlayer = player1;
                 return currentPlayer;
             }
         }
-        private static void initBoard(char[][] board)
+        private static void InitBoard(char[][] board)
         {
 
-            for (int i = 0; i < 10; i++)
+            for (int i = 0; i < numOfBoards; i++)
             {
-                board[i] = new char[9];
-                for (int j = 0; j < 9; j++)
+                board[i] = new char[cellsNumber];
+                for (int j = 0; j < cellsNumber; j++)
                 {
-                    board[i][j] = ' ';
+                    board[i][j] = empty;
 
                 }
             }
         }
 
-        private static void updateBoard(char[][] board)
+        private static void PrintBoard(int x, int y, char[][] board, int smallBoard)
```

Comment:
Naming: not clear what do 'x' and  'y' represent.. based on the code they are the top left coordinate of the board - better name for 'x' would be 'boardOriginLeft' and better name for 'y' would be 'boardOriginTop'